components/ws-daemon/pkg/iws/iws.go

```diff
@@ -934,7 +934,9 @@ func (wbs *InWorkspaceServiceServer) WorkspaceInfo(ctx context.Context, req *api
 
 	resources, err := getWorkspaceResourceInfo(wbs.CGroupMountPoint, cgroupPath, unified)
 	if err != nil {
-		log.WithError(err).Error("could not get resource information")
+		if !errors.Is(err, os.ErrNotExist) {
```

Comment:
@utam0k as an alternative, what do you think of `log.WithError(err).Warn("could not get resource information")`? If you were to do as `warn`, how does it show in [preview logs](https://www.notion.so/gitpod/Querying-logs-for-your-Preview-Environment-234e73f2e67c474c94f4389c1703d594)?

Reply:
The same thing happened with cpu limit, but there was too much warning logging when it was warn. Probably, this log would be logged at a very high rate when stopping the workspace. So I decided not to log it. WDYT?

Reply:
Was stopping also a factor with logging noise for cpu limit?

Assuming yes, and you did the same thing with CPU limit re: `os.ErrNotExist`, then it makes sense to be consistent.

Reply:
Yes, here
https://github.com/gitpod-io/gitpod/blob/b1e8f5d1e11b35ea2bb0aa0ff967b4f2186d7a3c/components/ws-daemon/pkg/cpulimit/dispatch.go#L120-L122

Reply:
Resolving :smile: